core/web/sessions_controller.go

```diff
@@ -83,6 +84,7 @@ func (sc *SessionsController) Destroy(c *gin.Context) {
 		return
 	}
 
+	sc.App.GetLogger().Audit(logger.AUTH_SESSION_DELETED, map[string]interface{}{"removedSessionID": sessionID})
```

Comment:
Would it make sense to say just `sessionID`, since `removed` is implied by the event, and then we can easily search for _all_ types of events associated with a single sessionID?
```suggestion
	sc.App.GetLogger().Audit(logger.AUTH_SESSION_DELETED, map[string]interface{}{"sessionID": sessionID})
```

Reply:
Yup good call, updated